cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/rolling/RollClient.java

```diff
@@ -115,7 +123,7 @@ interface RollClient {
      * @throws io.strimzi.operator.common.UncheckedExecutionException
      * @throws io.strimzi.operator.common.UncheckedInterruptedException
      */
-    Map<Integer, Configs> describeBrokerConfigs(List<NodeRef> toList);
+    Map<Integer, Configs> describeKafkaConfigs(List<NodeRef> toList);
```

Comment:
Should it be `describeNodeConfigs`? 

In any case you probably need to update the javadoc too, no?